### PR TITLE
add(bot): limit coauthor trailer whitespace

### PR DESCRIPTION
**Pull request by chdsbd**
If the commit message body is empty, don't add the two line spacing before coauthor trailers. GitHub adds it's own line between the title and body, so we'll have a three line space between the title and trailers if we don't remove our added spacing.

related #577

### CODE REVIEW

File: bot/kodiak/test_evaluation.py [hunk 1]
```diff
@@ -3294,7 +3294,16 @@ async def test_mergeable_include_coauthors() -> None:
     config = create_config()
     config.merge.message.include_coauthors = True
 
-    for body_style in (MergeBodyStyle.pull_request_body, MergeBodyStyle.empty):
+    for body_style, commit_message in (
+        (
+            MergeBodyStyle.pull_request_body,
+            "# some description\n\nCo-authored-by: Barry Block <73213123+b-block@users.noreply.github.com>",
```
Comment (sbdchd) [inline bot/kodiak/test_evaluation.py:3300]: I think we should have more tests with some tabs and other whitespace in the commit message

Comment (chdsbd) [reply bot/kodiak/test_evaluation.py:3300]: how about now that we're not trimming anything.

**Review by sbdchd — APPROVED** [review]
